v0/pipeline/steps/cd/k8s-dry-run-manifest-step-info.yaml

```diff
@@ -11,6 +11,8 @@ allOf:
       - type: string
         pattern: (<\+.+>.*)
         minLength: 1
+    encodeRawOutputYaml:
```

Comment:
What does Raw mean here?

Reply:
since we are removing all kind of redaction on manifest for this particular change,  and return the exact data which we are getting from kubectl. hence added raw in the name. 
disucessed with Anshul too for the name

Reply:
Can we name it at least encrypt and not encode?

Reply:
And I don't think it really make sense this Raw, this is an option for user to use and not describe our internal logic. The action is to encrypt step output